app/grandchallenge/products/models.py

```diff
@@ -22,6 +22,14 @@ class Company(models.Model):
         blank=True,
         help_text="Short summary of this project, max 250 characters.",
     )
+    slug = models.CharField(
```

Comment:
Django actually has a specific `SlugField` for this: https://docs.djangoproject.com/en/3.0/ref/models/fields/#slugfield

Reply:
There is also a handy `AutoSlugField` from django extensions: https://django-extensions.readthedocs.io/en/latest/field_extensions.html

This then allows you to use the shorthand:
```
slug = AutoSlugField(populate_from="company_name")
```
and then it will be automatically populated with the `company_name` field on the model.